Example is not working

When I run the example given in README I get:
```
error[E0599]: no function or associated item named `new` found for struct `FPSCounter` in the current scope
 --> src\main.rs:9:39
  |
9 |     let mut fps_counter = FPSCounter::new();
  |                                       ^^^ function or associated item not found in `FPSCounter`

error[E0599]: no function or associated item named `new` found for struct `App` in the current scope
  --> src\main.rs:10:24
   |
10 |     let mut app = App::new();
   |                        ^^^ function or associated item not found in `App`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `tetris-rust` due to 2 previous errors
```

The code is:
```
use ruscii::app::{App, State};
use ruscii::drawing::Pencil;
use ruscii::gui::FPSCounter;
use ruscii::keyboard::{Key, KeyEvent};
use ruscii::spatial::Vec2;
use ruscii::terminal::Window;

fn main() {
    let mut fps_counter = FPSCounter::new();
    let mut app = App::new();

    app.run(|app_state: &mut State, window: &mut Window| {
        for key_event in app_state.keyboard().last_key_events() {
            match key_event {
                KeyEvent::Pressed(Key::Esc) => app_state.stop(),
                KeyEvent::Pressed(Key::Q) => app_state.stop(),
                _ => (),
            }
        }

        fps_counter.update();

        let mut pencil = Pencil::new(window.canvas_mut());
        pencil.draw_text(&format!("FPS: {}", fps_counter.count()), Vec2::xy(1, 1));
    });
}

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,19 +1,23 @@
 ## Current 0.4.0 ([#12](https://github.com/lemunozm/ruscii/issues/12))
+
 - Removed most of the `new()` methods in order to use `default()`.
 - Added basic animations
 - Added `from()` methods for `Vec2` and `Direction`
 - Added `draw_right_aligned_text()` method.
-- Added `Vec<Vec<>>` indexing operators by `Vec2 `
-- Dependencies updated.
+- Added `Vec<Vec<T>>` indexing operators by `Vec2 `
+- Updated dependencies.
 
 ## Current 0.3.2
-- Improvement key support.
+
+- Improved key support.
 
 ## Current 0.3.1
-- Improvement key support with arrow keys.
-- Fixed issue with F keys.
+
+- Improved support for arrow keys.
+- Fixed issue with the F key.
 
 ## Release 0.3.0
+
 - Added `Debug` trait to `Color` and `Style`
 - Added `draw_at` and `draw` versions for `Drawable` elements.
 - Improved speed.
@@ -26,6 +30,7 @@
   The user should control this behaviour.
 
 ## Release 0.2.0
+
 - Added `get_keys_down` method from `keyboard` now gives an ordered time event list of keys.
 - Added `Drawable` trait for custom painting.
 - Added scale option to pencils.
@@ -34,10 +39,13 @@
   (This will be fixed in a near future)
 
 ## Release 0.1.2
+
 - Fixed issue with key events. Sometimes the key release event was not computed.
 
 ## Release 0.1.1
-- Added MacOS support
+
+- Added macOS support
 
 ## Release 0.1.0
+
 - First release

--- a/README.md
+++ b/README.md
@@ -106,8 +106,8 @@ use ruscii::spatial::{Vec2};
 use ruscii::gui::{FPSCounter};
 
 fn main() {
-    let mut fps_counter = FPSCounter::new();
-    let mut app = App::new();
+    let mut fps_counter = FPSCounter::default();
+    let mut app = App::default();
 
     app.run(|app_state: &mut State, window: &mut Window| {
         for key_event in app_state.keyboard().last_key_events() {

